catkin_tools/jobs/catkin.py

```diff
@@ -504,7 +504,7 @@ def create_catkin_build_job(context, package, package_path, dependencies, force_
             [MAKE_EXEC, 'install'],
             cwd=build_space,
             logger_factory=CMakeMakeIOBufferProtocol.factory,
-            locked_resource='installspace'
+            locked_resource=None if context.isolate_install else 'installspace'
```

Comment:
Possible this isn't necessary given that the underlying logic seems to swap out a `FakeLock`:

https://github.com/catkin/catkin_tools/blob/2442e1f971ae6bbd0c14fa6da183ee095ae099e3/catkin_tools/verbs/catkin_build/build.py#L535-L537

However, that's not turned on automatically when in isolated install mode, so maybe that's the better change to be making here?